sdk/metricsadvisor/ai-metrics-advisor/src/models.ts

```diff
@@ -1008,6 +1038,10 @@ export type MetricBoundaryCondition =
        * should be specified only when using other metric to filter
        */
       triggerForMissing?: boolean;
+      /**
+       * data used to implement value filter
+       */
+      type?: "Value" | "Mean";
```

Comment:
is this optional or non-optional?

Reply:
On the generated layer these are optional and readonly. Anu has asked on the service channel to clarify if they can have null values. Maybe we can wait till then.

Reply:
what `type` is this? Hard to guess its meaning. Please ask service team about it.